### PR TITLE
Feature/sg 544 docker docs

### PR DESCRIPTION
**Pull request by shaydeci**

### CODE REVIEW

File: documentation/source/SGDocker.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# SG Docker
```
Comment (ofrimasad) [inline documentation/source/SGDocker.md:1]: please change the title:
SuperGradients Docker Container

File: documentation/source/SGDocker.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Docker is an open-source containerization platform allowing developers to package and distribute applications in a portable and efficient way. Docker is becoming increasingly important in deep learning because it provides an easy and flexible way to manage the complex dependencies and configurations required for deep learning projects. With Docker, deep learning developers can easily package their applications and libraries into container images, which can be distributed and run on any machine with Docker installed. This simplifies the development process and makes it easier to reproduce and share deep learning experiments and results.
+
+## Instructions and Recommended Practices
+
+1) Install Docker https://docs.docker.com/engine/install/ubuntu/.
```
Comment (ofrimasad) [inline documentation/source/SGDocker.md:7]: We need Nvidia Docker. Don't we?

**Review by ofrimasad — COMMENTED** [review]
looks great!
minor comment inline

File: documentation/source/SGDocker.md (hunk 2 above)
Comment (shaydeci) [reply documentation/source/SGDocker.md]: nice catch, changed.

**Review by ofrimasad — APPROVED** [review]
LGTM

**Pull request merged**